docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -141,48 +131,48 @@ zk-2      0/1       Running   0         19s
 zk-2      1/1       Running   0         40s
 ```
 
-The StatefulSet controller creates three Pods, and each Pod has a container with 
+The StatefulSet controller creates three Pods, and each Pod has a container with
 a [ZooKeeper](http://www-us.apache.org/dist/zookeeper/stable/) server.
 
 ### Facilitating Leader Election
 
-As there is no terminating algorithm for electing a leader in an anonymous 
-network, Zab requires explicit membership configuration in order to perform 
-leader election. Each server in the ensemble needs to have a unique 
+As there is no terminating algorithm for electing a leader in an anonymous
```

Comment:
Does "As" here mean "Because"? Or "When"? Or "If"? (Seems like "Because," in which case please change/clarify)